net-p2p/nicotine+/nicotine+-3.3.6.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..13} )
+DISTUTILS_USE_PEP517=setuptools
+
+inherit distutils-r1 optfeature xdg
+
+DESCRIPTION="Graphical client for the Soulseek peer to peer network written in Python"
+HOMEPAGE="https://nicotine-plus.org/"
+SRC_URI="https://github.com/Nicotine-Plus/nicotine-plus/archive/${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/nicotine-plus-${PV}"
+
+LICENSE="GPL-3+"
```

Comment:
How is https://github.com/nicotine-plus/nicotine-plus/tree/master/pynicotine/external used in this version we package? It has at least MIT and CC-BY-SA-4.0 licenses being used.
https://github.com/nicotine-plus/nicotine-plus/blob/master/pynicotine/external/LICENSE.md